packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterHarp.yml

```diff
@@ -1,7 +1,7 @@
 languageId: plaintext
 command:
   version: 1
-  spokenForm: chuck after harp
+  spokenForm: chuck trailing harp
```

Comment:
This seems wrong

Reply:
This seems correct to me. We used to support chuck before and after and they did the same thing as chuck leading/trailing today.
https://github.com/cursorless-dev/cursorless/blob/6a78771b47308a8574a0fa46a50723c0f2c936e9/packages/cursorless-engine/src/core/commandVersionUpgrades/upgradeV1ToV2/upgradeV1ToV2.ts#L108-L114